pkg/portfwd/client.go

```diff
@@ -64,9 +58,6 @@ func HandleUDPConnection(ctx context.Context, client *guestagentclient.GuestAgen
 		buf := make([]byte, 65507)
 		for {
 			n, addr, err := conn.ReadFrom(buf)
-			if errors.Is(err, io.EOF) {
-				return nil
-			}
```

Comment:
I'm not sure about this change - https://pkg.go.dev/io#ReaderFrom is documented to not return io.EOF, but https://pkg.go.dev/net#PacketConn does not mention io.EOF, so we can get io.EOF.

I'll change this to handle the io.EOF inside the err != nil check.